src/main/java/com/everis/utils/IdConverter.java

```diff
@@ -0,0 +1,21 @@
+package com.everis.utils;
+
+import java.util.UUID;
+
+import javax.persistence.AttributeConverter;
+
+public class IdConverter implements AttributeConverter<UUID, String> {
```

Comment:
I think the class name is not really reflecting the real responsibility of this class. What about something like UUIDConverter ?